kokoro/angle/build.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# Copyright (C) 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to download and build ANGLE and all of its dependencies for AGI
+# Use this ANGLE branch for AGI (Note: keep it main until a release branch is created)
+
+ANGLE_BRANCH=main
+
+PYTHON_VERSION="3.9.5"
+
+APK_DIR="$KOKORO_ARTIFACTS_DIR/ANGLE/angle/out/APK"
+
+gn_args=(
+"android32_ndk_api_level = 26"
+"android64_ndk_api_level = 26"
+"angle_enable_annotator_run_time_checks = true"
+"angle_enable_trace = true"
+"angle_enable_vulkan = true"
+"angle_enable_vulkan_validation_layers = false"
+"angle_expose_non_conformant_extensions_and_versions = true"
+"angle_force_thread_safety = true"
+"angle_libs_suffix = \"_angle\""
+"dcheck_always_on = false"
+"enable_remoting = true"
+"ffmpeg_branding = \"Chrome\""
+"is_component_build = false"
+"is_debug = false"
+"is_official_build = true"
+"proprietary_codecs = true"
+"symbol_level = 2"
+"target_cpu = \"arm64\""
+"target_os = \"android\""
+)
+
+
+function setup_python_virtual_env {
+  pyenv uninstall -f "$1"
```

Comment:
Can we not use the pyenv in the system instead of reinstalling?

Reply:
I tried but Kokoro was throwing errors. The only workaround I was able to find was this.